health-checker/src/main/java/io/stargate/health/CheckerResource.java

```diff
@@ -6,7 +6,6 @@
 import com.codahale.metrics.health.HealthCheck;
 import com.codahale.metrics.health.HealthCheck.Result;
 import com.codahale.metrics.health.HealthCheckRegistry;
-import com.google.common.annotations.VisibleForTesting;
```

Comment:
As with earlier auth-modules, Guava usage in `health-checker` minimal so decided to pull out/replace this particular usage.